text/0044-heartbeat.md

```diff
@@ -0,0 +1,69 @@
+- Start Date: 2022-12-09
+- RFC Type: feature
+- RFC PR: N/A
+- RFC Status: draft
+
+# Summary
+
+Introduce an infrastructure monitoring agent in all SDKs supported by us. This agent will be triggered every time the user launches the application and will send us information about the state of their integration with Sentry.
```

Comment:
> Introduce an infrastructure monitoring agent

I think this sentence is making a decision on what the RFC is about, instead of focusing on the problem.

Reply:
Do you have any suggestion on how we could better describe it?

Reply:
the problem statement is something like this I think:

> Today Sentry is relies on receiving errors and transactions from SDKs to take users into the rest of their onboarding experience. This reliance seems to put a weighted importance on pushing the user to send a transaction or error to confirm they have done things correctly. Pushing them through an early process which is an opportunity loss for Sentry and the user to appreciate what they can do with the product.

I'd say that is more or less the problem. And the solution proposal or direction is something like we want SDKs to be able to send a functionally useful event/message which is not dependant on an error or transaction trigger from a specific application/service.